### PR TITLE
OP-507 Admin manual changes; remove all the 'you'

### PR DESCRIPTION
**Pull request by dbmalkovsky**
@mizzioisf once this is committed you can start making your changes

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -238,25 +238,25 @@ D:\OpenHospital_1_11_0\mysq> mysql –u root -p
...
 mysql>
 ----
 
-Create the database user with password, the main OH database and grant the necessary permissions with the following SQL commands (you can adjust database name, user and password according to your specific setup):
+Create the database user with password, the main OH database and grant the necessary permissions with the following SQL commands:
 
 [source,shell]
 ----
-mysql>  CREATE DATABASE oh; CREATE USER 'isf'@'localhost' IDENTIFIED BY 'isf123'; 
+mysql>  CREATE DATABASE isf; CREATE USER 'isf'@'localhost' IDENTIFIED BY 'isf123';
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]: again db 'isf'?

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -328,37 +328,37 @@ More information on these files can be found in the sections <<database-properti
 
 ==== 3.1.6 Open Hospital - Startup and run
 
-We are ready now to run the Open Hospital software by executing the proper launcher:
+The Open Hospital software is now ready to be run by executing the proper launcher command:
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc:331]:
```suggestion
The Open Hospital software is now ready to be ran by executing the proper launcher command:
```

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: Sorry; my merge was bad; fixed.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**